mitosheet/src/components/taskpanes/Download/DownloadTaskpane.tsx

```diff
@@ -21,6 +22,18 @@ import DefaultEmptyTaskpane from '../DefaultTaskpane/DefaultEmptyTaskpane';
 import DefaultTaskpaneFooter from '../DefaultTaskpane/DefaultTaskpaneFooter';
 
 
+const INVALID_CHARACTERS_IN_FILENAME = [
```

Comment:
Given that we define this constant, and the code that consumes it twice across both download taskpanes, how would you feel about moving this into a shared utility like `getInvalidFileNameError(fileName: string): string | undefined`?